src/DssDirectDepositAaveDai.sol

```diff
@@ -103,6 +103,11 @@ interface RewardsClaimerLike {
     function getRewardsBalance(address[] calldata assets, address user) external view returns (uint256);
```

Comment:
```suggestion
```

Reply:
Actually this will break as it is being used in the test. But I agree we shouldn't leave interfaces in the main code for the test. Will decouple them.